superset/assets/visualizations/nvd3_vis.js

```diff
@@ -345,8 +345,32 @@ function nvd3Vis(slice, payload) {
       svg = d3.select(slice.selector).append('svg');
     }
     if (vizType === 'dual_line') {
-      chart.yAxis1.tickFormat(d3.format(fd.y_axis_format));
-      chart.yAxis2.tickFormat(d3.format(fd.y_axis_2_format));
+      const yAxisFormatter1 = d3.format(fd.y_axis_format);
+      const yAxisFormatter2 = d3.format(fd.y_axis_2_format);
+      chart.yAxis1.tickFormat(yAxisFormatter1);
+      chart.yAxis2.tickFormat(yAxisFormatter2);
+      chart.useInteractiveGuideline(true);
+      chart.interactiveLayer.tooltip.contentGenerator(function (d) {
+        const tooltipTitle = xAxisFormatter(d.value);
+        let tooltip = '';
+
+        tooltip += "<table><thead><tr><td colspan='3'>"
```

Comment:
wondering if we can generalize this tooltip markup in order to share with tooltips for other vis's....